README.md

```diff
@@ -1 +1,105 @@
+
+## Commands
+
+### edamame init
+
+Usage: `edamame init`
+Outputs:
+
+```
+⠋ Configuring EBS Credentials...Please enter a password to associate with your Edamame Grafana dashboard & Postgres database account.
+Password: {password}
+Password for PG & Grafana has been set as:
+{password}
+✔ Cluster configured. You can load test now!
+```
+
+- Creates an EKS cluster using the associated AWS account (~15 mins)
+- Configures the permissions and credentials needed to manage that cluster.
+- Sets up the operator that will manage the distributed load test
+- Sets up the database
+- Sets up Grafana, which will be used to visualize load test metrics.
+
+**Note**: it can as long as *20 minutes* to create the cluster and apply the necessary permissions.
+
+### edamame run
+
+Usage: `edamame run --file {/path/to/test.js} --vus {max number of VUs in test run}`
+Outputs:
+
+```
+⠇ Running k6 load test...Polling load generators...
+⠧ Running k6 load test...Assessing load generators; counting the number completed
+Number of load generators completed: 0
+⠹ Running k6 load test...Polling load generators...
+⠇ Running k6 load test...Assessing load generators; counting the number completed
+Number of load generators completed: 0
+⠴ Running k6 load test...Polling load generators...
+⠹ Running k6 load test...Assessing load generators; counting the number completed
+Number of load generators completed: 0
+⠏ Running k6 load test...Polling load generators...
+⠼ Running k6 load test...Assessing load generators; counting the number completed
+Number of load generators completed: 0
+⠸ Running k6 load test...Polling load generators...
+⠏ Running k6 load test...Assessing load generators; counting the number completed
+Number of load generators completed: 0
+⠦ Running k6 load test...Polling load generators...
+⠼ Running k6 load test...Assessing load generators; counting the number completed
+Number of load generators completed: 4
+✔ Removed load generating components from cluster.
+```
+
+- Automatically scales cluster nodes to handle the amount of VUs specified in the test file
+- Runs the test
+- Automatically deletes and scales down the cluster once test is completed.
+
+**Notes**:
+
+- The `--file` flag takes the relative path of the test script you want to run. This should be written in JavaScript, and follow conventions of k6 testing scripts. See [here](https://k6.io/docs/examples/) for examples and tutorials on how to write a k6 test script.
+- The `--vus` flag is going to be removed soon, and `edamame` will read the max number of VUs directly from the provided test script.
+- To run a sample test, use the sample test file provided in the `k6_tests` directory. From the project's root dir, the relative path will be `./k6_tests/test.js`
+
+### edamame get
+
+Usage: `edamame get`
+Outputs:
+
+```
+✔ Retrieving all test ids...
+[{"id":1},{"id":2},{"id":3}]
+```
+
+- Retrieves all the ids for tests that have been run in JSON format.
```

Comment:
Can you make a note that this feature will also be deprecated in favor of test names or randomly generated uuids?

Reply:
@lukeoguro  Do you mean the entire feature of getting the ids will be removed, or just that the format of the ids may change (to user generated test names/UUIDs)?